### PR TITLE
Fixes to deploy-ovm-pair

### PR DESCRIPTION
**Pull request by K-Ho**
1. Quick fix for `node publish deploy-ovm-pair` - The L1 deploy was trying to deploy `MockToken.sol` which is only compiled if we have the `test-helper` flag set to true in the `build` command
2. Pass in L1 and L2 endpoints as command options so that we can run this deploy against testnet as well as locally

### CODE REVIEW

**Review by i-stam — APPROVED** [review]
lgtm

File: publish/src/commands/deploy-ovm-pair.js [hunk 1]
```diff
@@ -29,16 +25,16 @@ const deployOvmPair = async () => {
...
-	const { l1Messenger, l2Messenger } = await getMessengers();
+	const { l1Messenger, l2Messenger } = await getMessengers({ dataProviderUrl });
 
 	await commands.connectBridge({
 		l1Network: 'local',
 		l2Network: 'local',
-		l1ProviderUrl: L1_PROVIDER_URL,
-		l2ProviderUrl: L2_PROVIDER_URL,
+		l1ProviderUrl: l1ProviderUrl,
+		l2ProviderUrl: l2ProviderUrl,
```
Comment (ajsantander) [inline publish/src/commands/deploy-ovm-pair.js:37]: nit: replace with 
```
l1ProviderUrl,
l2ProviderUrl
```

**Review by ajsantander — APPROVED** [review]
Looks good! Although I only see changes related to adding new parameters to the `deploy-ovm-pair` script.

I don't see anything related to the compilation of MockToken.

**Comment by i-stam** [comment]
> Looks good! Although I only see changes related to adding new parameters to the `deploy-ovm-pair` script.
> 
> I don't see anything related to the compilation of MockToken.

I think adding this parameter when using build script solves the problem:  `testHelpers: true` ,(deploy-ovm-pair.js, line 49).

File: publish/src/commands/deploy-ovm-pair.js (hunk 1 above)
Comment (tynes) [reply publish/src/commands/deploy-ovm-pair.js]: Good call, this is addressed

**Comment by tynes** [comment]
Once this is merged, I can update https://github.com/ethereum-optimism/docker/pull/9 to use the `develop` branch and then it will easily work with the integrations repo. I have a script locally similar to `up.sh` that will also run the `deploy-ovm-pair` script, planning on adding it to the integrations repo when all of this stuff is merged.

**Review by justinjmoses — APPROVED** [review]

**Comment by K-Ho** [comment]
Closing in favor of https://github.com/Synthetixio/synthetix/pull/1034

**Pull request closed without merging**